source/clear-linux/tutorials/azure.rst

```diff
@@ -1,125 +1,481 @@
-          --os-type Linux \
-          --image-urn clear-linux-project:clear-linux-os:${offering}:${clrversion}.0.0 \
-          --nic-name clr-nic1 \
-          --vnet-name ${clrversion}-${offering}-vnet \
-          --vnet-address-prefix 10.0.0.0/8 \
-          --vnet-subnet-name ${clrversion}-vnet-subnet \
-          --vnet-subnet-address-prefix 10.0.0.0/8 \
-          --plan-name ${offering} \
-          --plan-publisher clear-linux-project \
-          --plan-product clear-linux-os \
-          --public-ip-domain-name ${azuser}-${clrversion}-${offering}-${uniquesuffix} \
-          --public-ip-allocation-method "Dynamic" \
-          --public-ip-name "${clrversion}-${offering}-${uniquesuffix}"
-
-
-
-SSH Sessions
-============
-
-To keep SSH sessions to Clear Linux Guests in Azure from being dropped
-after periods of inactivity, you can give the following option to SSH via
-the command line::
-
-   -o ServerAliveInterval=180
-
-Alternatively, you can add this setting to your SSH config file as shown
-below::
-
-   Host *:
-      ServerAliveInterval 180
-
-.. [1]
-   Software and workloads used in performance tests may have been optimized
-   for performance only on Intel microprocessors. Performance tests are
-   measured using specific computer systems, components, software, operations
-   and functions. Any change to any of those factors may cause the results to
-   vary. You should consult other information and performance tests to assist
-   you in fully evaluating your contemplated purchases, including the
-   performance of that product when combined with other products. For more
-   complete information, visit http://www.intel.com/performance/datacenter.
-   Configuration: Clear Linux OS release 11130 on SKU Standard_DS3_v2 in
-   Microsoft\* Azure\*.
+#. Create a |CL| virtual machine.
+
+#. Log into the |CL| VM and use it.
+
+#. Stop the |CL| VM.
+
+#. Deallocate resources to stop incurring charges.
+
+Install MS Azure CLI 2.0 on Clear Linux
+***************************************
+
+Prerequisites for using the MS Azure CLI 2.0 on your |CL| system require you
+to have Python 2.7 or later, libffi, and OpenSSL 1.0.2 installed. If you need
+to install these packages on your |CL| machine, install the sysadmin-basic
+software bundle using the :command:`swupd` command:
+
+.. code-block:: console
+
+   sudo swupd bundle-add sysadmin-basic
+
+.. note::
+
+   These instructions are for installing the MS Azure CLI 2.0 tools on a |CL|
+   system. If you are installing the CLI on another platform, follow the
+   instructions in the `MS Azure Install Azure CLI tutorial`_ for your
+   specific operating system.
+
+To install the MS Azure CLI 2.0 on |CL|, use the :command:`curl` command as
+shown:
+
+.. code-block:: console
+
+   curl -L https://aka.ms/InstallAzureCli | bash
+
+If you get an error message from :command:`curl` related to the -L parameter,
+or an error message is generated that includes the text "Object Moved", use
```

Comment:
> +If you get an error message from :command:`curl` related to the -L parameter,
> +or an error message is generated that includes the text "Object Moved", use

Maybe it would be good to show example of the actual error message?

Reply:
Actually, I'm having heartburn about this whole process. I got this example straight from the MS Azure tutorials and blindly downloading a script from a website and piping it into a shell to execute is dangerous to say the least. I could download it into a file and have the user manually look at it and then execute it as a script.